xdem/spatialstats.py

```diff
@@ -47,13 +49,17 @@
 from skimage.draw import disk
 
 from xdem._typing import NDArrayf
+from xdem.misc import deprecate
 
 with warnings.catch_warnings():
     warnings.filterwarnings("ignore", category=DeprecationWarning)
     import skgstat as skg
 
 
-def nmad(data: NDArrayf | RasterType, nfact: float = 1.4826) -> np.floating[Any]:
+@deprecate(
+    removal_version=Version("0.4"), details="xDEM's nmad will be replaced by the new nmad implementation in GeoUtils"
```

Comment:
I think we should point them directly towards the new signature so they don't have to look for it, something along the lines of: "xdem.spatialstats.nmad is being deprecated in favor of geoutils.stats.nmad"